### PR TITLE
Join visits

### PR DESCRIPTION
**Pull request by KaskMIL**
# In this new Pull Request I'm adding the following features:
- Create a Vets table and populate with data.
- Create a join table "specializations" to relation vets and species table.
- Create a join table "visits" to relation vets and animals table.
- Answer the question given using the following queries:
   - Who was the last animal seen by William Tatcher?

Captura de Pantalla 2022-08-12 a la(s) 14 39 10

   - How many different animals did Stephanie Mendez see?

Captura de Pantalla 2022-08-12 a la(s) 14 45 33

   - List all vets and their specialties, including vets with no specialties.

Captura de Pantalla 2022-08-12 a la(s) 14 53 17

   - List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.

Captura de Pantalla 2022-08-12 a la(s) 15 00 38

   - What animal has the most visits to vets?

Captura de Pantalla 2022-08-12 a la(s) 15 21 01

   - Who was Maisy Smith's first visit?

Captura de Pantalla 2022-08-12 a la(s) 15 28 15

   - Details for most recent visit: animal information, vet information, and date of visit.

Captura de Pantalla 2022-08-12 a la(s) 15 50 55

   - How many visits were with a vet that did not specialize in that animal's species?

Captura de Pantalla 2022-08-12 a la(s) 18 34 06

   - What speciality should Maisy Smith consider getting? Look for the species she gets the most.

Captura de Pantalla 2022-08-12 a la(s) 18 14 27

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -35,4 +35,21 @@ SELECT species.name, COUNT(*) FROM animals JOIN species ON animals.species_id =
...
+
+SELECT vets.name AS "Vet Name", animals.name AS "Animal Name", visits.date FROM vets JOIN visits ON vets.id = visits.vet_id JOIN animals ON animals.id = visits.animal_id WHERE vets.name LIKE '%William Tatcher%' GROUP BY vets.name, animals.name, visits.date ORDER BY visits.date DESC LIMIT 1;
+SELECT COUNT(animals.name) AS "Stephanie Mendez's Animals" FROM animals JOIN visits on animals.id = visits.animal_id JOIN vets ON vets.id = visits.vet_id WHERE vets.name LIKE '%Stephanie Mendez%';
+
+SELECT vets.name AS "Vet Name", species.name AS "specialization" FROM vets FULL OUTER JOIN specializations ON vets.id = specializations.vets_id FULL OUTER JOIN species ON species.id = specializations.species_id GROUP BY vets.name, species.name;
+
+SELECT animals.name AS "Animal Name", visits.date AS "Date" FROM animals JOIN visits ON animals.id = visits.animal_id JOIN vets ON vets.id = visits.vet_id WHERE vets.name LIKE '%Stephanie Mendez%' AND visits.date BETWEEN '2020-04-01' AND '2020-08-30' GROUP BY animals.name, visits.date;
+
```
Comment (tufoinnkuo10) [inline queries.sql]: - [x] Great job so far. The queries work as expected, but they give difficulties to read. Kindly add comments to the queries, identifying the query questions you are answering. Great work 👍🏾

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @KaskMIL 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
- good use of screenshots

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by KaskMIL** [comment]
Hi @tufoinnkuo10 
Thanks for your review, I will implement the changes
Happy coding!

**Review by algerina — APPROVED** [review]
Hi @KaskMIL , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**